### DISCUSSION TITLE
路由器翻墙后无法使用 Pyncmd 音源，之前是不翻用不了现在反过来了

### DISCUSSION THREAD

**Discussion by miyukisakurai** [1715173329/UnblockNeteaseMusic#920] [Q&A] [answered]
测试关掉路由器翻墙后正常，请问音源是不是屏蔽非大陆ip了？
可以问问ip或者域名么，我想设置个例外

**Comment by 1715173329** [comment, marked as answer]
https://github.com/UnblockNeteaseMusic/server/commit/4fe40100f89f7547f2eff33ac84adbd3a8afe5a3

**Reply by miyukisakurai** [reply]
谢谢，添加例外解决了
